docs-new/docs/graphql/core/api-reference/schema-metadata-api/index.mdx

```diff
@@ -0,0 +1,243 @@
+   "args": <args-object>
+}
+```
+
+### Request body
+
+<div class="parsed-literal">
+
+[Query](#api-query)
+
+</div>
+
+#### Query {#api-query}
+
+| Key     | Required | Schema     | Description                     |
+|---------|----------|------------|---------------------------------|
+| type    | true     | String     | Type of the query               |
+| args    | true     | JSON Value | The arguments to the query      |
+| version | false    | Integer    | Version of the API (default: 1) |
+
+## Request types
+
+The various types of queries are listed in the following table:
+
+| `type`                                                                                                                                                        | `args`                                                                                                                                                  | `version` | Synopsis                                                         |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|------------------------------------------------------------------|
+| **bulk**                                                                                                                                                      | [Query](#api-query) array                                                                                                                               | 1         | Execute multiple operations in a single query                    |
+| [run_sql](/graphql/core/api-reference/schema-metadata-api/run-sql.mdx#schema-metadata-run-sql)                                                                | [run_sql_args](/graphql/core/api-reference/schema-metadata-api/run-sql.mdx#schema-metadata-run-sql-args)                                                                | 1         | Run SQL directly on Postgres                                     |
+| [track_table](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-track-table)                                                     | [TableName](/graphql/core/api-reference/syntax-defs.mdx#tablename)                                                                                      | 1         | Add a table/view                                                 |
+| [track_table](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-track-table-v2)                                                  | [track_table_args](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-track-table-v2-syntax)                                                | 2         | Add a table/view with configuration                              |
```

Comment:
hash param mismatch for `track_table_args`

pls fix - `#schema-metadata-track-table-syntax-v2`

Reply:
Done